### ISSUE TITLE
Scanln fails to stop for input during debug on Linux

### ISSUE THREAD

**Issue by michaljaros84** [derekparker/delve#679]
1. What version of Delve are you using (`dlv version`)?
 0.11.0-alpha

2. What version of Go are you using? (`go version`)?
1.7.3

3. What operating system and processor architecture are you using?
Ubuntu 16.04.1 LTS 64 bit

4. What did you do?
Trying to get program to stop for input on fmt.Scanln

5. What did you expect to see?
Execution to wait for input when Scanln is reached

6. What did you see instead?
Execution continues and program acts like empty string/white space was entered

7. Additional info:
- tested on 2 different machines with same OS and Go version - same issue
- tested on Windows 10 with same Go version - no issue

Hello, 

Long story short: looks like Scanln throws EOF when called through the dlv (both in command line and Intelij IDE)

Originally raised that in go-lang-plugin-org/go-lang-idea-plugin but this seems to be delve issue. Please see: https://github.com/go-lang-plugin-org/go-lang-idea-plugin/issues/2838 for more details.

Anyway project is great: finally I can debug Go code without bunch of fmt.Println flying around.

Cheers,
Michal

**Comment by aarzilli** [comment]
Duplicate of #65.
